personal_assistant/contacts/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from .forms import ContactsForm
+from .models import Contact
+
+def main(request):
+    contacts = Contact.objects.all()
+    return render(request, 'contacts/index.html', {"contacts": contacts})
+
+def add_contact(request):
+
+    if request.method == 'POST':
+        form = ContactsForm(request.POST)
+        if form.is_valid():
+            new_contact = form.save()
```

Comment:
Змінна `new_contact` ніде не використовується тож можна і без неї:
```suggestion
            form.save()
```